### DISCUSSION TITLE
Telegram Bot ConversationHandler Skips Waiting for User Input and Gives AttributeError 'NoneType' object has no attribute 'reply_text'

### DISCUSSION THREAD

**Discussion by startfrom0sen** [python-telegram-bot/python-telegram-bot#4456] [Q&A]
I'm building a Telegram bot that interacts with the user to edit and upload videos using the python-telegram-bot library. The bot retrieves videos from Reddit, edits them using moviepy, and then asks the user for additional input through a conversation handler. However, I'm encountering a couple of issues:

1. AttributeError: 'NoneType' object has no attribute 'reply_text': I receive this error when the bot tries to call reply_text on update.message.reply_text("test")

2. Conversation Skipping Input: When I use update.effective_message.reply_text (which isnt None) to ask a question, the bot skips waiting for the user's answer and starts re-uploading the next video in the loop.

Here's the link for the stack overflow post (full breakdown): https://stackoverflow.com/q/78949957/27195691

I'm stuck on this problem and thought I would give it a shot to also post it here. I really dont know how to solve this issue...

Thank you very much!

**Comment by Bibo-Joshi** [comment]
Please don't ask the same question in multiple places at the same time 

https://stackoverflow.com/questions/78949957/telegram-bot-conversationhandler-skips-waiting-for-user-input-and-gives-attribut
